integrationtests/wildfly-modules/src/test/java/org/infinispan/test/integration/as/InfinispanStoreJdbcIT.java

```diff
@@ -58,11 +63,31 @@ public void testCacheManager() {
             .timestampColumnType("BIGINT")
          .dataSource().jndiUrl("java:jboss/datasources/ExampleDS");
 
-      EmbeddedCacheManager cm = new DefaultCacheManager(gcb.build(), builder.build());
-      Cache<String, String> cache = cm.getCache();
-      cache.put("a", "a");
-      assertEquals("a", cache.get("a"));
-      cm.stop();
+      EmbeddedCacheManager cm = null;
+      try {
+         cm = new DefaultCacheManager(gcb.build(), builder.build());
+
+         Cache<String, String> cache = cm.getCache();
+         cache.put("a", "a");
+         assertEquals("a", cache.get("a"));
+      } finally {
+         if (cm != null)
+            cm.stop();
+      }
+   }
+
+   @Test
+   public void testXmlConfig() throws IOException {
+      EmbeddedCacheManager cm = null;
+      try {
+         cm = new DefaultCacheManager("jdbc-config.xml");
+         Cache<String, String> cache = cm.getCache("anotherCache");
+         cache.put("a", "a");
```

Comment:
Should we verify the cache store is actually present? Using something like

`
PersistenceManager clm = TestingUtil.extractComponent(cache, PersistenceManager.class);
assertEquals(1, clm.getStores(JdbcStringBasedStore.class).size());
`